Makefile

```diff
@@ -227,13 +227,14 @@ ZAPD       := tools/ZAPD/ZAPD.out
 FADO       := tools/fado/fado.elf
 PYTHON     ?= $(VENV)/bin/python3
 
-# Command to replace path variables in the spec file. We can't use the C
-# preprocessor for this because it won't substitute inside string literals.
-SPEC_REPLACE_VARS := sed -e 's|$$(BUILD_DIR)|$(BUILD_DIR)|g'
+# Command to replace $(BUILD_DIR) in some files with the build path.
+# We can't use the C preprocessor for this because it won't substitute inside string literals.
+BUILD_DIR_REPLACE := sed -e 's|$$(BUILD_DIR)|$(BUILD_DIR)|g'
 
 # Audio tools
 AUDIO_EXTRACT := $(PYTHON) tools/audio_extraction.py
 SAMPLECONV    := tools/audio/sampleconv/sampleconv
+SBC           := tools/audio/sbc --matching
```

Comment:
it feels wrong to have an argument for the command here. should it be tied to the main `NON_MATCHING` switch above? looking at usage it seems to just be adding some unnecessary junk that's only needed to match, is that correct?

Reply:
Not supplying --matching isn't properly tested so I haven't added anything to toggle it automatically yet (especially applicable to other tools like `sfc` coming later). I think that should wait until later. You're right though, in this program in particular its only use is for the buffer bug garbage data processing.

Reply:
Thoughts about still splitting off the `--matching` to a `SBC_FLAGS` now even without auto-toggling it? Understandable if you want to wait to do that as well.

Reply:
Split it into `SBCFLAGS`